diwako_cbrn.vr/scripts/cbrn/functions/fn_threatPFH.sqf

```diff
@@ -94,6 +96,14 @@ if (_hasChemDetector isNotEqualTo (_player getVariable ["cbrn_detector_beeps", f
     };
 };
 
+if (_hasGeigerCounter isNotEqualTo (_player getVariable ["cbrn_detector_geiger", false])) then {
+	_player setVariable ["cbrn_detector_geiger", _hasGeigerCounter];
+	if (cbrn_geiger && {cbrn_geigerPfh < 0}) then {
+        cbrn_geigerPfh = [cbrn_fnc_detectorGeigerPFH, 0.05, [cba_missiontime]] call CBA_fnc_addPerFrameHandler;
+    };
+};
+
+
```

Comment:
What is the point of this, to kick start the geiger counter after some event? Isn't it enough to activate/deactivate it on user action?

Reply:
This is just a 1:1 copy from the beeper, which is being kickstarted exactly the same way :)

Reply:
I can't believe i write bad code... moving, no one saw anything